doc/api/modules.md

```diff
@@ -368,7 +368,7 @@ example, then `require('./some-library')` would attempt to load:
 
 <!--type=misc-->
 
-If the module identifier passed to `require()` is not a native module,
+If the module identifier passed to `require()` is not a core module,
```

Comment:
The word "core" isn't defined, I would use the word "built-in", its more descriptive, and more accurate.

Btw:

> All core modules are native but not all native modules are in core.

This is not true. There are native modules built-in to node, but there are also lots of javascript modules built-in to node, see https://github.com/nodejs/node/tree/master/lib, and user's of the Node.js API can't tell the difference between native and javascript built-in modules, to my knowledge.